app/assets/stylesheets/mtl/extend/_buttons.scss

```diff
@@ -0,0 +1,8 @@
+.btn-icon {
```

Comment:
These two are very different. The icon is a squared button, matching the pixel height of 36 from the variables. So how about this?

```CSS
.btn.square {
  padding: 0;
  width: $button-height;
  text-align: center;
}
.btn.slim {
  padding: 0 10px;
}
```